python/README.md

```diff
@@ -0,0 +1,31 @@
+Generate a virtual environment with, for example,
+
+```
+$ mkvirtualenv syntax
```

Comment:
Now that virtual environments have been in the standard library for a long time (don't tell Jython!), let's use that


```suggestion
```bash
$ python -m venv syntax
```

Reply:
Well, I think I copied the instructions from Nammu :sweat_smile: